lib/fastlane/plugin/wpmreleasetoolkit/actions/common/get_prs_list_action.rb

```diff
@@ -6,34 +6,19 @@ module Actions
     class GetPrsListAction < Action
       def self.run(params)
         repository = params[:repository]
-        start_tag = params[:start_tag]
-        end_tag = params[:end_tag]
-        report_path = params[:report_path]
+        report_path = File.expand_path(params[:report_path])
+        tag = params[:tag]
```

Comment:
Since this PR already introduce backward incompatible changes, may I suggest to rename this option to "milestone" to better describe what using it does?

```suggestion
        milestone = params[:milestone]
```

Reply:
Oh, I see that there's actually already [a new `:milestone` option in the `ConfigItem` array below](https://github.com/wordpress-mobile/release-toolkit/pull/227/files#diff-c3c92f31a3aed155aab1aba9315c428344a5079f853257ac495f27c81c12b317R53-R56).

I'm guessing is just something that got lost during the implementation, then.

~~I also guess it went unnoticed because of Fastlane's dynamic nature and the fact that if you pass it an unexpected option (`tag` in this instance, as the PR description suggests) it will still use it.~~

In fact, if I run the lane using `tag` as suggested in the description, it fails:

```
[10:53:44]: You passed invalid parameters to 'get_prs_list'.
[10:53:44]: Check out the error below and available options by running `fastlane action get_prs_list`

[!] Could not find option 'tag' in the list of available options: repository, report_path, milestone
```

Running the same command with `milestone` instead of `tag` fails, too, but that's because there's no match for `params[:tag]`

```
[!] Could not find option 'tag' in the list of available options: repository, report_path, milestone
```

With the suggestions I left, basically `%s/tag/milestone/`, the code works.

Reply:
Fixed in 8c3e4affab2acde6067ba4af4de8d044591a350d, thanks!!